README.md

```diff
@@ -1,8 +1,8 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# WEBPACK 5 TEMPLATE
+# Leaderboard: setup project
 
-> Everything you need to start using webpack is all setup and ready to use just run the following commands to get started:
+> This is a simple plain html leaderboard set up project:
 
 Get started by copying and pasting the following in your terminal: 
 
```

Comment:
- [ ] Please update the cloning link in you Readme from Webpack to Leaderboard, the same thing for the command `cd webpack`, Please change it to `cd Leaderboard` so that your Readme is professional.
- [ ] Please remove the text below the `npm start` or change it with the proper one. Your app will not show `Hello webpack!`.
- [ ] Your app runs on localhost port: 8080. Please update the link from `http://localhost:3000` to `http://localhost:8080`.